src/com/n8pickle/puzzle/Board.java

```diff
@@ -54,7 +95,9 @@ public boolean equals(Object y)        // does this board equal y?
 
     public Iterable<Board> neighbors()     // all neighboring boards
     {
-        return new MyIterator<E>();
+        Stack s = new Stack();
+        ...
```

Comment:
?

Reply:
This was asked to be done this way in the assignment instructions. It will be finished soon